app/components/forms/admin/content/pages-form.js

```diff
@@ -79,7 +79,7 @@ export default Component.extend(FormMixin, {
   actions: {
     submit(data) {
       this.onValid(() => {
-        this.sendAction('save', data);
+        this.save(data);
```

Comment:
`Uncaught TypeError: _this.save is not a function`

![Screenshot from 2021-01-12 16-49-56](https://user-images.githubusercontent.com/56407566/104308150-3b638d00-54f6-11eb-903d-78446f2abb34.png)